qcelemental/util/misc.py

```diff
@@ -91,22 +91,42 @@ def filter_comments(string):
     return string
 
 
-def unnp(dicary, path=None):
-    """Return `dicary` with any ndarray values replaced by lists."""
-    # TODO handle complex numbers/arrays
+def unnp(dicary, flat=False, _path=None):
+    """Return `dicary` with any ndarray values replaced by lists.
 
-    if path is None:
-        path = []
+    Parameters
+    ----------
+    dicary: dict
```

Comment:
Should flat default to True as this is expected in schema?

Reply:
defaulting to `flat=True` complicates fragments, since that's properly nested. I keep it as ndarray internally for handy arithmetic.